app/assets/stylesheets/new.scss

```diff
@@ -0,0 +1,28 @@
+#border {
+  border: 2px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.